themes/fast-track-theme/lms/templates/ccx/edit_ccx_form.html

```diff
@@ -48,9 +49,8 @@
         <div class="field inline">
             <label class="sr" for="ccx_delivery_mode">${_('Delivery Mode')}</label>
             <select name="delivery_mode" id="ccx_delivery_mode" data-current-value="${ccx.delivery_mode}">
-                <option value="In Person - Required">In Person - Required</option>
-                <option value="In Person - Optional">In Person - Optional</option>
-                <option value="Online Only">Online Only</option>
+                <option value="IN_PERSON">In Person</option>
```

Comment:
Change values. Maybe it would be better not to hardcode this but pull from model.